docs/pages/admin-guides/access-controls/access-requests/automatic-reviews.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Configure Automatic Reviews
+description: Describes how to configure Access Monitoring Rules for Automatic Reviews.
+---
+
+Teleport supports automatic reviews of role Access Requests. This
+feature enables teams to enforce a zero standing privilege policy, while
+still allowing users to receive temporary access without manual approval.
+
+Support for automatic reviews of resource Access Requests is currently in
+development.
+
+## How it works
+
+Automatic reviews are triggered by Access Monitoring Rules. These rules instruct
+Teleport to monitor specific Access Requests and automatically submit a review
+when certain conditions are met. The conditions can be configured based on the
+requested roles, and based on the requesting user's Teleport traits.
```

Comment:
Nit: I would rephrase this a bit so we don't have to change this when we add more conditions.
```suggestion
when certain conditions (such as requested roles or user traits) are met.
```